app/src/main/kotlin/com/wire/android/ui/markdown/MDList.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.ui.markdown
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.buildAnnotatedString
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.theme.wireTypography
+import org.commonmark.node.BulletList
+import org.commonmark.node.Document
+import org.commonmark.node.ListBlock
+import org.commonmark.node.Node
+import org.commonmark.node.OrderedList
+
+@Composable
+fun MDBulletList(bulletList: BulletList, nodeData: NodeData) {
+    val marker = "\u2022"
```

Comment:
Maybe we should move things like this also to the style? I mean, it would be cool to be able to change the bullet into something else just in one place in the code depending on the style we want to have 😄

Reply:
Extracted it to markdown consts